src/macros.jl

```diff
@@ -752,7 +751,7 @@ macro defVar(args...)
     end
 
     sdp = any(t -> (t == :SDP), extra)
-    symmetric = sdp | any(t -> (t == :Symmetric), extra)
+    symmetric = sdp || any(t -> (t == :Symmetric), extra)
```

Comment:
Would help to add parenthesis around the condition here. I can't tell the precedence of ``=`` versus ``||``.